.github/workflows/prerequisite-test.yml

```diff
@@ -0,0 +1,14 @@
+name: Verify there is matching build of security plugin
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    name: Run prerequisite tests
```

Comment:
Similar to above: maybe "Run prerequisite _checks_"?

Reply:
Thanks! I have changed the name for both test's name and the sub tests' name. Please review it.